docs/quickstart.md

```diff
@@ -252,6 +267,7 @@ Group=harvest
 Type=simple
 Restart=on-failure
 ExecStart=${HARVEST_INSTALL_PATH}/harvest-${HARVEST_VERSION}-1_linux_amd64/bin/harvest --config ${HARVEST_INSTALL_PATH}/harvest-${HARVEST_VERSION}-1_linux_amd64/harvest.yml start -f %i
```

Comment:
It's customary to put WorkingDirectory above ExecStart although it should work fine below.

What about using this instead?
WorkingDirectory=${HARVEST_INSTALL_PATH}/harvest-${HARVEST_VERSION}-1_linux_amd64
ExecStart=./bin/harvest start -f %i